examples/vanilla/control-elements/media-error-dialog.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Media Error Dialog</title>
+    <script type="module" src="../../../dist/index.js"></script>
+    <script type="module">
+      import { CustomVideoElement } from 'https://cdn.jsdelivr.net/npm/custom-media-element/+esm';
+
+      class CustomVideo extends CustomVideoElement {
+        #error;
+
+        get error() {
+          return this.#error ?? super.error;
+        }
+
+        set error(value) {
+          this.#error = value;
+        }
+      }
+
+      globalThis.customElements.define('custom-video', CustomVideo);
+    </script>
+
+    <style>
+      /* Hide custom elements that are not defined yet */
+      :not(:defined) {
```

Comment:
nitpick(non-blocking): This should probably be applied specifically to `media-error-dialog` for what we're demonstrating, right? Or is there a reason you hide every custom element for this demo?